cmd/node/app_test.go

```diff
@@ -329,8 +330,14 @@ func txWithRunningNonceGenerator(dependencies []int) TestScenario {
 				time.Sleep(250 * time.Millisecond)
 				actNonce = getNonce()
 			}
-			tx, err := types.GenerateCallTransaction(acc1Signer, dst, uint64(i), 10, 1, 1)
-			suite.NoError(err, "failed to create signed tx: %s", err)
+			var tx *types.Transaction
+			if i == 0 {
+				tx = transaction.GenerateSpawnTransaction(acc1Signer, dst)
```

Comment:
I'm wondering how this can work... If the spawn transaction is basically empty and it is the only transaction that gets the nonce 0 and accounts are still initialized with nonce == 0 and skipping is not allowed. How can any transaction make it in?

Reply:
What do you mean by "skipping is not allowed"? If the nonce is 0, a spawn TX is generated here, then nonce gets increased inside https://github.com/spacemeshos/go-spacemesh/blob/feature/move-tests-tx-gen-into-svm/cmd/node/app_test.go#L344, so next TXs become call TXs